### PR TITLE
Dashboard improvements

### PR DESCRIPTION
**Pull request by Kinzeng**
This PR improves the dashboard that comes out of the box with the ClickHouse instance. Instead of showing a long Code: 516 error message in each chart, we show a friendlier error message at the top.

The PR also fixes a bug where deleting charts and then editing the other charts doesn't clear the plots correctly.

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: programs/server/dashboard.html [hunk 1]
```diff
@@ -318,10 +327,16 @@
 
 /// If it is hosted on server, assume that it is the address of ClickHouse.
 if (location.protocol != 'file:') {
-    host = location.origin;
+    // host = location.origin;
+    host = 'https://pig7cys591.us-west-2.aws.clickhouse-dev.com';
```
Comment (Felixoid) [inline programs/server/dashboard.html:331]: Is it left from development?

Comment (Kinzeng) [reply programs/server/dashboard.html]: ah yeah sorry, just removed it

**Review by alexey-milovidov — APPROVED** [review]

**Comment by alexey-milovidov** [comment]
1. The parameter controls should also be hidden.
2. Red font on light blue looks weird:
Screenshot_20230202_162629

**Comment by alexey-milovidov** [comment]
I also want it to display an "invitation" to enter the credentials - instead of an error.
We can move the controls for "Address", "User", "Password" to the center of the page, and add labels - so it will look like a modal dialog.

**Pull request merged**